src/app/code/community/Shopgate/Framework/Model/System/Config/Source/Cart/Types.php

```diff
@@ -28,24 +28,12 @@ public function toOptionArray()
     {
         return array(
             array(
-                'value' => ShopgateClient::TYPE_MOBILESITE,
-                'label' => Mage::helper('shopgate')->__('Mobile site')
+                'value' => Shopgate_Framework_Helper_Client::VALUE_APP,
+                'label' => Mage::helper('shopgate')->__('Shopgate Mobile App'),
             ),
             array(
-                'value' => ShopgateClient::TYPE_IPHONEAPP,
-                'label' => Mage::helper('shopgate')->__('IPhone App')
```

Comment:
this is a breaking change, you will need to traverse configs and re-assign all these values to existing ones. Else, I think it might actually throw a hard error on their admin page. Best case scenario all their mobile coupons become free for all :D

Reply:
Nope, they all get invalidated.
So the old app only coupons are not usable anymore, which is what the PO wanted